README.rst

```diff
@@ -354,6 +354,8 @@ List of supported endpoints
 
     # Covid-19 Area Report
```

Comment:
Thanks @siddydutta , it all looks good! Just a small suggestion to make it clear for the users. 
In line 355, could you rename the "Covid-19 Area Report" to "Travel Restrictions v1" and line 357 the  "Covid-19 Report" to "Travel Restrictions v2"? Since the official name of the API is Travel Restrictions it will be easier to identity if that way and also distinguish the different versions.

Reply:
Yes, that makes sense. Updated the commit. @tsolakoua